packages/mgt-element/src/providers/IProvider.ts

```diff
@@ -26,20 +27,39 @@ export abstract class IProvider implements AuthenticationProvider {
    * @memberof IProvider
    */
   public graph: IGraph;
+
   /**
-   * Enable/Disable multi account functionality
+   * Specifies if the provider supports multiple account functionality
    *
    * @protected
    * @type {boolean}
    * @memberof IProvider
    */
   protected isMultipleAccountSupported: boolean = false;
+
+  /**
+   * Enable/Disable multi account functionality
+   *
+   * @protected
+   * @type {boolean}
+   * @memberof IProvider
+   */
+  protected isMultipleAccountEnabled: boolean = false;
+
+  /**
+   * Specifies if Multi account functionality is supported by the provider and enabled.
+   *
+   * @readonly
+   * @type {boolean}
+   * @memberof IProvider
+   */
+  public get isMultiAccountSupportedAndEnabled(): boolean {
+    return this.isMultipleAccountSupported && this.isMultipleAccountEnabled;
+  }
```

Comment:
We could simplify this to a single getter for a provider to override. The enabled disabled properties/logic can be just in the Msal2Provider.

```suggestion
  /**
   * Specifies if Multi account functionality is supported by the provider and enabled.
   *
   * @readonly
   * @type {boolean}
   * @memberof IProvider
   */
  public get isMultipleAccountsSupportedAndEnabled(): boolean {
    return false;
  }
```